icra23_experiments:
  loop_rate_hz: 5.0 # 30.0
  num_experiments: 6 # 20 # 60 # 70 # 30 # 40
  random_seed: 1 # 395487 # 99 # 801 # 669 # 3234
  output_dir: /home/ellis/astrobee/src/ellis/ellis_planner/output
  time_limit_sec: 900.0 # 15 minutes
  start_at_index: 57 # 56 # 46 # 0
  run_method:
    ours: True
    replanning: False
    cmax: True

  start:
    pose:
      x: -0.45
      y: 0.65
      yaw: 0.0
  randomize: False

  goal:
    pose:
      x: 0.7
      y: 0.65
      yaw: 0.0
    randomize: False

  robot_collision_box:
    size:
      x: 0.32
      y: 0.32

  obstacles:
    - type: rectangle
      name: obs1
      x: 0.0
      y: 0.0
      yaw: 0.0
      size_x: 0.28
      size_y: 0.28
      known: True
      randomize: True
      randomization_size_x: [-0.05, 0.25]
      randomization_size_y: [-0.05, 0.20]

  disturbances:
    - center_x: 0.0
      center_y: 0.5
      size_x: 0.4
      size_y: 1.0
      # force_x: -0.15
      # force_y: -0.35
      force_x: 0.0
      force_y: -0.35
      randomize: True
      randomization_force_angle: [-0.85, -0.25] # [-0.75, 0.0] # [-0.75, 0.10] # [-0.10, 0.50] # [-0.25, 0.25]
      randomization_force_mag: [0.0, 0.15] # [0.0, 0.25] # [-0.05, 0.25] # [-0.15, 0.50] # [-0.15, 0.15]

obstacle_manager:
  freq: 10.0
  world_frame_id: world
  gazebo_ns: /gazebo
  topics:
    vis: /mob/ellis_planner/vis
  obstacles: []

disturbance_manager:
  world_frame_id: world
  body_frame_id: body
  gazebo_ns: /gazebo
  topics:
    vis: /mob/ellis_planner/vis
  disturbances: []
